jhotdraw-gui/src/main/java/org/jhotdraw/gui/JDisclosureToolBarBeanInfo.java

```diff
@@ -150,45 +150,31 @@ public int getDefaultEventIndex() {
    * @return An image object representing the requested icon. May return null if no suitable icon is
    *     available.
    */
+  public java.awt.Image loadImage(String iconName, java.awt.Image icon) {
+    if (iconName == null) {
+      return null;
```

Comment:
Since `iconName` is necessary, throwing an exception could be considered. I understand that it was done to keep the behavior or because it's not a real error. It's just something to consider.